lib/Minify/ScssCssSource.php

```diff
@@ -0,0 +1,175 @@
+<?php
+use Leafo\ScssPhp\Compiler;
+use Leafo\ScssPhp\Server;
+use Leafo\ScssPhp\Version;
+
+/**
+ * Class for using SCSS files
+ *
+ * @link https://github.com/leafo/scssphp/
+ */
+class Minify_ScssCssSource extends Minify_Source {
```

Comment:
Should it be Minify_ScssCssSource or Minify_ScssSource? If Minify_ScssCssSource, shouldn't it extend Minify_CssSource?